### DISCUSSION TITLE
Feedback on new docs

### DISCUSSION THREAD

**Discussion by JonathanSchmidt1** [lab-cosmo/upet#138] [General]
uncertainty supporting models are not mentioned in https://lab-cosmo.github.io/upet/latest/models.html is this intentional
UPET_UQ_SUPPORTED_MODELS = [
    "pet-mad-s-v1.0.2",
    "pet-mad-xs-v1.5.0",
    "pet-mad-s-v1.5.0",
]
neither dos
PET-MAD DOS
PET_MAD_DOS_LATEST_STABLE_VERSION = "1.0"
PET_MAD_DOS_AVAILABLE_VERSIONS = ["1.0"]

**Comment by JonathanSchmidt1** [comment]
should this be fixed:
metatomic/torch/ase_calculator.py:1482: UserWarning: `compute_requested_neighbors_from_options` is deprecated and will be removed in a future version. Please use `neighbor_lists_for_model` to get the calculators and call them directly.
  vesin.metatomic.compute_requested_neighbors_from_options(

**Reply by abmazitov** [reply]
This should be fixed on a metatomic level, but please feel free to just add this to the ignore

**Reply by Luthaf** [reply]
This will be fixed by the next release of metatomic_ase

**Comment by JonathanSchmidt1** [comment]
Should we have examples for e.g. DOS, uncertainty?

**Reply by sofiia-chorna** [reply]
same for fine-tuning, the examples are either in atomistic-cookbook or metatrain

**Reply by JonathanSchmidt1** [reply]
same for gromacs lammps etc examples being prepared right now

**Reply by abmazitov** [reply]
I think that in general it would be super-nice to make the cookbook examples on PET embedded into the current docs somehow, I just don't know if that is technically possible with sphinx

**Reply by Luthaf** [reply]
If nothing else, we should be able to add links to the tutorial through [intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html)

**Reply by abmazitov** [reply]
> If nothing else, we should be able to add links to the tutorial through [intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html)

I think that's a brilliant idea, I'd be happy to see how it works

**Reply by sofiia-chorna** [reply]
so do we want it to be embedded or just a link with [intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html)?

**Reply by abmazitov** [reply]
Ideally - embedded, but that can be hard to achieve. Otherwise I see it as if we had neat links with preview thumbnails that lead to atomistic cookbook

**Comment by abmazitov** [comment]
> uncertainty supporting models are not mentioned in https://lab-cosmo.github.io/upet/latest/models.html is this intentional UPET_UQ_SUPPORTED_MODELS = [ "pet-mad-s-v1.0.2", "pet-mad-xs-v1.5.0", "pet-mad-s-v1.5.0", ] neither dos PET-MAD DOS PET_MAD_DOS_LATEST_STABLE_VERSION = "1.0" PET_MAD_DOS_AVAILABLE_VERSIONS = ["1.0"]

That's a good catch, I didn't want to mention the availability of UQ there, but I agree that it might be nice to mention this list somewhere near the UQ block in the docs https://lab-cosmo.github.io/upet/latest/usage/ase.html#uncertainty-quantification.

**Comment by sandrasaade** [comment]
For the Usage, ASE interface, I am not sure exactly about the pre-trained sketch-map dimensionality reduction, I feel it is not clear enough. Is this shown in the code snippet below?

**Reply by sandrasaade** [reply]
Discussed with Sofiia, perhaps worth linking the example in chemiscope, otherwise IMO it may be confusing

**Reply by abmazitov** [reply]
Yeah, I agree, the current example is quite minimalistic. Please feel free to adapt it and make it more clear! I would be also very much in favor of embedding the chemiscope here!

**Comment by JonathanSchmidt1** [comment]
I noticed that usage completely replaces examples and adresses my issues. can we just remove examples?

**Reply by abmazitov** [reply]
Shall we remove the current examples entirely and replace them with embedded parts of the atomistic cookbook, as @Luthaf has suggested above?

**Reply by Luthaf** [reply]
That would be ideal IMO, this way we don't have to maintain multiple copies of the same thing

**Comment by sandrasaade** [comment]
Another question regarding the Batched evaluation with metatrain, Step 2: It mentions the dataset in extxyz format, however in the code snippet the dataset is passed as an .xyz file?

**Reply by abmazitov** [reply]
Yes, this is normal, ASE treats `xyz` as `extxyz`. We mention extxyz just because the plain xyz doens't allow to store anything but positions

**Comment by Luthaf** [comment]
The docs seems to be missing API documentation, ideally generated with autodoc to have all functions/submodules/classes visible!

**Reply by sofiia-chorna** [reply]
fair point, happy to fix it

**Reply by abmazitov** [reply]
Yes, please! Good catch!

**Reply by sofiia-chorna** [reply]
fixed in https://github.com/lab-cosmo/upet/pull/140

**Comment by sandrasaade** [comment]
For the Frequently Asked Questions: When mentioning XS, XL or S model, it would be worth stating somewhere the list of hyperparameters associated with each model size, and usually what kind of applications they are suitable for. Otherwise, I have not heard about these labels before

**Reply by abmazitov** [reply]
Maybe we can put the link to Filippo's paper and say that the full list of hypers available here [https://arxiv.org/abs/2601.16195](https://arxiv.org/abs/2601.16195) - it's in the Section 5 of the SI.

**Reply by sandrasaade** [reply]
Yep exactly, that would work as well!

**Reply by abmazitov** [reply]
Alternatively we can just put this table in the docs, maybe that will be even better

**Reply by JonathanSchmidt1** [reply]
Pr is open https://github.com/lab-cosmo/upet/pull/139

**Reply by abmazitov** [reply]
> Pr is open #139

Approved

**Comment by GardevoirX** [comment]
This page is not linked in the new doc, it's intended?
https://github.com/lab-cosmo/upet/blob/df94f29600d6bdcde75ccd3b4a09006f66d61100/docs/SPEED.md

**Reply by sofiia-chorna** [reply]
i think it would be nice to have a speed page in the docs

**Reply by abmazitov** [reply]
Yes, very good idea! In general we should probably add some scripts to replicate this plot on a local hardware, but for now let's add it to the docs

**Reply by GardevoirX** [reply]
Here #141
